src/markdown-pages/docs/running-simbank-tests/web-app-integration-test.md

```diff
@@ -0,0 +1,162 @@
+---
+path: "/docs/running-simbank-tests/web-app-integration-test"
+title: "WebAppIntetgrationTest"
+---
+
+The `WebAppIntetgrationTest` is somewhat different to the previously described Galasa SimBank tests as it shows how you can use Galasa to test a hybrid cloud application that uses a mix of platforms and technologies. A mix of technologies makes end-to-end integration testing complicated. Use the `WebAppIntetgrationTest` to help understand how Galasa simplifies integration testing in such an environment.
+
+The `WebAppIntetgrationTest` test demonstrates the end-to-end integration testing of an application that runs on different platforms (z/OS and Cloud) and uses different technologies (a 3270 emulator, JCL batch job and Selenium Web Driver).
+
+The following diagram shows how you can use the WebAppIntegrationTest to test a hybrid cloud application: 
+
+![Galasa in hybrid cloud](../../galasa-hybrid-cloud.svg)
+
+The diagram highlights the use of the following technology, and the interaction between the test and those technologies.
+
+ - The 3270 emulator creates a random account number and checks that it does not exist in the CICS region
+ - If the account does not exist the account is created by the Z/OS Batch Manager and is inserted into the backend database
+ - Docker then generates a web page
+ -  Selenium drives the web browser to complete a web form and confirm the balance of the account
+
+
+## Running the WebAppIntegrationTest
+
+As the `WebAppIntetgrationTest` is slightly different to the other tests, there's a bit more set up to do than for the other supplied SimBank tests. 
+
+The test is still run locally but is designed to help you to understand how to set the properties to enable the test to run in a Galasa Ecosystem. When running locally, you can use the localhost setting `127.0.0.1` for increased security. 
+
```

Comment:
using 127.0.0.1 has nothing to do with security

Reply:
reworded